e2e-tests/web/test/specs/navigateToDashboard.e2e.ts

```diff
@@ -1,12 +1,11 @@
 import { URL } from 'url'
 
-import { defaultCity, filter, language, Routes } from '../../../shared/constants'
+import { defaultCity, filter, Routes } from '../../../shared/constants'
 import LandingPage from '../pageobjects/landing.page'
 
 describe('navigate to dashboard', () => {
   it('filter and navigate to City', async () => {
     const dashboardPath = Routes.dashboard
-    LandingPage.language = language
```

Comment:
Not sure about this one but `LandingPage` does not have a `language` attribute and it seems not to be used.

Reply:
@Taggotty seems like you wrote this test, any information?